### PR TITLE
Copyedit Part 1 content

### PR DESCRIPTION
**Pull request by shingjay**
## Checklist

1. Closes issue: 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: content/part1.md [hunk 1]
```diff
@@ -154,11 +157,11 @@ func main() {
...
 ```
 
-## Adding and Removing Elements from the Cart
+## Adding and removing elements from the cart
 
-In order to support adding and removing elements from the cart, the workflow needs to respond to different types of signals.
-Signals let you send arbitrary signals to workflows.
-The below code listens to a signal channel for messages that either add or remove items from a shopping cart.
+In order to support adding and removing elements from the cart, the Workflow needs to respond to different types of signals.
+Signals let you send arbitrary signals to Workflows.
```
Comment (shingjay) [inline content/part1.md:163]: > Signals let you send arbitrary signals to Workflows.
This doesn't sound right - did you mean something else? A signal channel?

Comment (sw-yx) [reply content/part1.md:163]: ha, very good catch. cc @vkarpov15

File: content/part1.md [hunk 2]
```diff
@@ -1,23 +1,26 @@
-Temporal promises to help you build invincible apps.
-To make this possible, Temporal introduces new design patterns that are very different from the traditional web app architecture.
+## Introduction
 
-Instead of API endpoints that talk to a database over the network, your API endpoints instead call in-memory _Workflows_ that store state internally.
-Temporal handles persisting the state of your Workflows and distributing your Workflow between Workers as necessary.
-You as the developer are responsible for implementing [Workflows](https://docs.temporal.io/docs/go/workflows) and [Activities](https://docs.temporal.io/docs/go/activities) as normal Go code, Temporal handles the data persistence and horizontal scaling for you.
+Temporal promised to help you build invincible apps.
```
Comment (flossypurse) [inline content/part1.md:3]:
```suggestion
Temporal promises to help you build invincible apps.
```
Why past tense?

File: content/part1.md [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
+To make this possible, we have introduced new design patterns that are very different from the ones used in a traditional web app architecture.
```
Comment (flossypurse) [inline content/part1.md:4]:
```suggestion
To make this possible, new design patterns are introduced that are very different from the ones used in a traditional web app architecture.
```

File: content/part1.md [hunk 4]
```diff
@@ -101,7 +104,7 @@ func main() {
 }
 ```
 
-In order to see this shopping cart workflow in action, you can create a _starter_ that sends queries and signals
+In order to see this shopping cart Workflow in action, you can create a _starter_ that sends queries and signals
 to modify the shopping cart.
```
Comment (flossypurse) [inline content/part1.md:108]: Put this on the same line as the sentence it is apart of.

File: content/part1.md [hunk 5]
```diff
@@ -154,11 +157,11 @@ func main() {
...
 
-## Adding and Removing Elements from the Cart
+## Adding and removing elements from the cart
 
-In order to support adding and removing elements from the cart, the workflow needs to respond to different types of signals.
-Signals let you send arbitrary signals to workflows.
-The below code listens to a signal channel for messages that either add or remove items from a shopping cart.
+In order to support adding and removing elements from the cart, the Workflow needs to respond to different types of signals.
+Signals let you send arbitrary signals to Workflows.
+The following code listens to a signal channel for messages that either add or remove items from a shopping cart.
```
Comment (flossypurse) [inline content/part1.md:164]:
```suggestion
The following code listens to a Signal channel for messages that either add or remove items from a shopping cart.
```

File: content/part1.md [hunk 6]
```diff
@@ -233,9 +236,9 @@ func RemoveFromCart(state *CartState, item CartItem) {
...
 
-## Next Up
+## Next up
 
-Temporal introduces a new way of building web applications: instead of storing a shopping cart in a database, you
-can represent a shopping cart as a long-lived Workflow.
-For simple CRUD applications like this shopping cart app so far, this pattern doesn't make things significantly easier.
-Next up, we'll look at a case where Temporal's long-lived Workflows shine: sending a reminder email if the user abandons their cart.
+Temporal introduces a new way of building web applications; instead of storing a shopping cart in a database, you
+can represent a shopping cart as a long-living Workflow.
```
Comment (flossypurse) [inline content/part1.md:242]: Why newline?

**Review by flossypurse — COMMENTED** [review]
Overall looking good - 
In order to take the next steps, can we put up a draft PR of the first blog post in the docs repo?
That will make it easy to add and curate the Snipsync placeholders.

File: content/part1.md (hunk 1 above)
Comment (vkarpov15) [reply content/part1.md:163]: Yeah this sentence is awkward. What do you think of "Signals are a way to notify Workflows of external events"?

File: content/part1.md (hunk 6 above)
Comment (vkarpov15) [reply content/part1.md:242]: Mistake on my part, will fix :+1:

File: content/part1.md (hunk 1 above)
Comment (shingjay) [reply content/part1.md:163]: Yeah that sounds better, @vkarpov15.

**Comment by shingjay** [comment]
Thanks for the review, @flossypurse. I've addressed them - let me know if I'm missing anything else, thanks!

**Review by vkarpov15 — APPROVED** [review]

**Pull request merged**